### PR TITLE
feat(release): npm release through jenkins

### PR DESCRIPTION
**Pull request by dmgauthier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,79 +1,110 @@
...
-  },
-  "prettier": {
-    "printWidth": 100,
-    "singleQuote": true,
-    "trailingComma": "all"
-  },
-  "jest": {
-    "setupTestFrameworkScriptFile": "<rootDir>/jestSetup.js"
-  }
+    "name": "coveo-enzyme-redux",
```
Comment (GermainBergeron) [inline package.json:2]: Thoughts about using `@coveord/enzyme-redux` instead?

File: package.json [hunk 2]
```diff
@@ -1,79 +1,110 @@
...
-  "keywords": [
-    "redux",
-    "enzyme",
-    "store",
-    "test"
-  ],
-  "engines": {
-    "node": ">=6"
-  },
-  "author": "Dennis Axelsson <knegusen@gmail.com>",
```
Comment (GermainBergeron) [inline package.json:22]: I would probably keep author and add coveo as a contributor? https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors

File: package.json (hunk 1 above)
Comment (dmgauthier) [reply package.json:2]: Will do !

File: package.json (hunk 2 above)
Comment (dmgauthier) [reply package.json:22]: That would be more honest.  I will certainly do that.

**Review by j-mcgregor — APPROVED** [review]

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    extends: [require.resolve('./eslint-config')],
```
Comment (gdostie) [inline .eslintrc.js:2]: We could install [tsjs](https://github.com/coveord/tsjs) as dependency instead and use its eslint-config instead?

File: .npmignore [hunk 4]
```diff
@@ -8,4 +8,6 @@ src
 
 .idea
 
+.prettierignore
```
Comment (gdostie) [inline .npmignore:11]: Why add this to the npmignore?

File: package.json [hunk 5]
```diff
@@ -1,79 +1,116 @@
...
+            "type": "MIT"
+        }
+    ],
+    "babel": {
+        "presets": [
+            "@babel/env",
+            "@babel/react"
+        ]
+    },
+    "prettier": {
```
Comment (gdostie) [inline package.json:104]: Use prettier config from tsjs ?

File: .eslintrc.js (hunk 3 above)
Comment (dmgauthier) [reply .eslintrc.js:2]: I though about it. but since there is no typescript in the repo, do you think it is okay still ?

File: .npmignore (hunk 4 above)
Comment (dmgauthier) [reply .npmignore:11]: IDK I thought that everything not usefull for a NPM module should go there

File: package.json (hunk 5 above)
Comment (dmgauthier) [reply package.json:104]: Well if TSJS do not depends on a TS version to be installed I'll do that

**Comment by aiheon** [comment]
Just saying Danny but I think you can close this? 😂

**Pull request closed without merging**